documentation/modules/ref-sample-cruise-control-config.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// assembly-deployment-configuration.adoc
+
+[id='ref-sample-cruise-control-config-{context}']
+= Sample Cruise Control YAML configuration
+
+For help in understanding the configuration options available for your Cruise Control deployment, refer to sample YAML file provided here.
+
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  # ...
+  cruiseControl:
+    image: my-org/my-image:latest <1>
+    replicas: 1 <2>
+    config: <3>
+      default.goals: >
+         com.linkedin.kafka.cruisecontrol.analyzer.goals.RackAwareGoal,
+         com.linkedin.kafka.cruisecontrol.analyzer.goals.ReplicaCapacityGoal
+      cpu.balance.threshold: 1.1
+      metadata.max.age.ms: 300000
+      send.buffer.bytes: 131072
+    resources: <4>
+      requests:
+        cpu: 200m
+        memory: 64Mi
+      limits:
+        cpu: 500m
+        memory: 128Mi
+    logging: debug <5>
+    template: <6>
+      pod:
+        metadata:
+          labels:
+            label1: value1
+        imagePullSecrets:
+          - name: my-docker-credentials
+        securityContext:
+          runAsUser: 1000001
+          fsGroup: 0
+        terminationGracePeriodSeconds: 120
+    readinessProbe: <7>
+      initialDelaySeconds: 15
+      timeoutSeconds: 5
+    livenessProbe: <8>
+      initialDelaySeconds: 15
+      timeoutSeconds: 5
+# ...
+----
+<1> ADVANCED OPTION: Container image configuration, which is xref:assembly-configuring-container-images-deployment-configuration-kafka[recommended only in special situations].
+<2> Number of application instances, can be set to 1 to run or 0 to pause the application.
+<3> Config specifies the Cruise Control configuration. xref:ref-cruise-control-configuration-{context}[Standard Cruise Control configuration may be provided, restricted to those properties not managed directly by {ProductName}].
+<4> xref:assembly-resource-limits-and-requests-deployment-configuration-kafka[CPU and memory resources to reserve].
+<5> Logging configuration, to log messages with a given severity (debug, info, warn, error, fatal) or above.
+<6> xref:assembly-customizing-deployments-str[Customization of deployment templates and pods].
+<7> xref:assembly-healthchecks-deployment-configuration-kafka[Healthcheck readiness probes].
+<8> xref:assembly-healthchecks-deployment-configuration-kafka[Healthcheck liveness probes].
```

Comment:
isn't this example already provided? I know modular documentation is the hell ... and tbh always confusing and difficult to review for me! ;-)